DJHelper/DJHelper/View Controllers/CreateEventViewController.swift

```diff
@@ -170,32 +149,39 @@ extension CreateEventViewController {
         eventController.authorize(event: event) { (results) in
             switch results {
             case let .success(eventRep):
-
-                // save the eventID that is returned to the newly created event object.
-                self.event?.eventID = eventRep.eventID ?? 999
-                try? CoreDataStack.shared.mainContext.save()
-
                 print("successful attempt to create event in vc: \(eventRep.name)")
                 DispatchQueue.main.async {
                     self.navigationController?.popViewController(animated: true)
+                    self.activityIndicator(shouldStart: false)
```

Comment:
Does this code get hit with the view controller being popped from the stack right before it?